src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -246,8 +246,13 @@ public static function get_theme_data( $deprecated = array(), $options = array()
 		$options = wp_parse_args( $options, array( 'with_supports' => true ) );
 
 		if ( null === static::$theme || ! static::has_same_registered_blocks( 'theme' ) ) {
-			$theme_json_data = static::read_json_file( static::get_file_path_from_theme( 'theme.json' ) );
-			$theme_json_data = static::translate( $theme_json_data, wp_get_theme()->get( 'TextDomain' ) );
+			$theme_json_file = static::get_file_path_from_theme( 'theme.json' );
```

Comment:
This change is fine. I can't think of a reason why this would break existing behavior.